packages/office-ui-fabric-react/src/components/Rating/Rating.scss

```diff
@@ -0,0 +1,85 @@
+@import '../../common/common';
+
+$defaultColor: $ms-color-neutralTertiary;
+$selectedColor: $ms-color-black;
+$hoverColor: $ms-color-themeDarkAlt;
+$disabledColor: $ms-color-neutralTertiaryAlt;
+
+// Hide input field
+.ms-Rating-input {
+  position: absolute;
+  opacity: 0;
+  top: 0px;
+}
+
+.ms-Rating-container {
```

Comment:
Ah no, there was a reason.. I want the overall control to be a block-level element, but for tracking hover for all the stars, I need an `inline-block` element wrapping them.